src/panels/Menu/types.ts

```diff
@@ -1,5 +1,6 @@
 export interface TaskbarItemConfig {
   id: string;
+  name: string; // The name of the menu item. This will show up in the OpenSpace launcher.
```

Comment:
Move this comment to on top of the variable. A change like that is incoming for the other variables as well :)